### PR TITLE
PR: Expose package version in SpyderShell

### PR DESCRIPTION
**Pull request by impact27**
Expose spyder kernels version for checking from the frontend

### CODE REVIEW

File: spyder_kernels/console/shell.py [hunk 1]
```diff
@@ -51,6 +52,8 @@ def __init__(self, *args, **kwargs):
         super(SpyderShell, self).__init__(*args, **kwargs)
         self._allow_kbdint = False
         self.register_debugger_sigint()
+        # Used for checking correct version by spyder
```
Comment (ccordoba12) [inline spyder_kernels/console/shell.py:55]:
```suggestion

        # Used for checking correct version by spyder
```

For clarity.

**Review by ccordoba12 — COMMENTED** [review]
@impact27, besides my tiny suggestion below I'd like to ask you: the improvements based on this PR are meant for master of both Spyder-kernels and Spyder, right?

**Comment by impact27** [comment]
> @impact27, besides my tiny suggestion below I'd like to ask you: the improvements based on this PR are meant for master of both Spyder-kernels and Spyder, right?

Yes the corresponding PR is https://github.com/spyder-ide/spyder/pull/19074, opened on master

**Review by ccordoba12 — APPROVED** [review]
Thanks @impact27!

**Pull request merged**